src/shell/dotnet/Shell/WebWindow.xaml.cs

```diff
@@ -59,7 +59,7 @@ private void TrySetIconUrl(WebWindowOptions webWindowOptions)
             return;
 
         // TODO: What's the default URL if the app is running from a manifest? We should probably not allow relative urls in that case.
```

Comment:
If for any reason the URL is not set, then we're defaulting to about:blank.

Reply:
Not my code :)